pattern-library/sass/patterns/_tables.scss

```diff
@@ -23,3 +24,148 @@ $table-border-color:                        palette(grayscale, x-light) !default
 // ----------------------------
 // #GLOBAL
 // ----------------------------
+
+
+%highlight {
+  background-color: palette(warning, x-trans);
```

Comment:
Should this use the informational-based color now? If for visual/color-specific reasons its a no, that's cool. Just wanted to check.